docs/use-service.md

```diff
@@ -17,68 +17,85 @@ Once you have your Mapzen API key you'll need include it with Terrain Tile reque
 Request a single tile with this URL pattern to get started:
 
 ```
-https://tile.mapzen.com/mapzen/terrain/v1/{format}/{z}/{x}/{y}.{extension}?api_key=your-mapzen-api-key
+https://tile.mapzen.com/mapzen/terrain/v2/{format}/{z}/{x}/{y}.{extension}?api_key=your-mapzen-api-key
```

Comment:
Yea, I think we should continue with `v1` on the `tile.mapzen.com` URLs.

This might be a touch confusing if we keep old data in S3; we had talked about copying existing data to a `v1` prefix. The new build would go in the same bucket with a `v2` prefix. Users who were using S3 directly would continue using old data until they changed their URLs to use the `v2` prefix while the folks that used Mapzen Terrain Tiles service would get `v2` data without changing anything.